packages/svelte/src/internal/client/dom/elements/bindings/input.js

```diff
@@ -115,6 +127,19 @@ export function bind_group(inputs, group_index, input, get_value, update) {
 	queue_micro_task(() => {
 		// necessary to maintain binding group order in all insertion scenarios. TODO optimise
 		binding_group.sort((a, b) => (a.compareDocumentPosition(b) === 4 ? -1 : 1));
+		if (hydration_mismatch) {
+			var value;
+			if (is_checkbox) {
+				value = get_binding_group_value(binding_group, value, input.checked);
+			} else {
+				var hydration_input = binding_group.find((input) => input.checked);
+				if (hydration_input !== undefined) {
+					// @ts-ignore
+					value = hydration_input.__value;
+				}
```

Comment:
We can code golf this a bit
```suggestion
				// @ts-ignore
				value = hydration_input?.__value;
```